main.tex

```diff
@@ -105,28 +105,28 @@ \subsubsection{Block-space efficiency}
+We present WabiSabi, a generalization of Chaumian CoinJoin based on a Keyed-Verification Anonymous Credentials-based (KVAC) scheme~\cite{chase2019signal}. The use of KVACs replaces blind signatures' standard denominations with homomorphic amount commitments, similar to Confidential Transactions~\cite{maxwell2016confidential}, but retains the invariant that the sum of any one participant's outputs does not exceed that of their inputs without the coordinator learning the underlying values. In addition to being more flexible this improves privacy compared to standard denominations, since smaller inputs can be combined and change outputs created with the same unlinkability guarantees as the privacy enhanced outputs\footnote{Note that the cleartext amounts appearing in the final transaction might still link individual inputs and outputs.}.
 
-WabiSabi can be instantiated to construct a variety of CoinJoin transaction structures that depart from the standard output denomination convention, such as SharedCoin\footnote{\url{https://github.com/sharedcoin/Sharedcoin}} and CashFusion\footnote{\url{https://github.com/cashshuffle/spec}} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Additionally it makes possible arbitrary consolidation of outputs, minimizing unmixed change, relaxing minimum required denominations, improved block space efficiency, making payments from CoinJoins, as well as payments within CoinJoins, also known as PayJoins.
+WabiSabi can be instantiated to construct a variety of CoinJoin transaction structures that depart from the standard output denomination convention, used by SharedCoin\footnote{\url{https://github.com/sharedcoin/Sharedcoin}} and CashFusion\footnote{\url{https://github.com/cashshuffle/spec}} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Additionally it makes possible arbitrary consolidation of outputs, minimizes unmixed change, relaxes minimum required denominations, improves block space efficiency, and makes payments from as well as within CoinJoins, also known as PayJoins, possible.
 
 \section{Preliminaries}
 
 Hereby we give an informal and high-level description of applied cryptographic primitives. In the following the security parameter is denoted as $\lambda$.
 
 \subsection{Commitment schemes}
-A commitment scheme allows a party to commit to a message without enabling them to change their mind about the committed message after publishing the commitment. On the other hand the commitment should not reveal anything about the committed message.
+A commitment scheme allows one to commit to a chosen message while preventing them from changng the message after publishing the commitment. Thee commitment should not reveal anything about the chosen message.
```

Comment:
```suggestion
A commitment scheme allows one to commit to a chosen message while preventing them from changng the message after publishing the commitment. These commitment should not reveal anything about the chosen message.
```

Reply:
This should be: `These commitments` or `The commitment`.